civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -433,6 +433,29 @@ function civihr_employee_portal_update_7033() {
   TaxonomyHelper::createDefaultTerms();
 }
 
+/**
+ * Revert updated features components
+ * - Panel page which holds the My Details page
+ * - Output Filter to write pure HTML in views sections without filters
+ * Refresh the node files as onboarding form node was altered
+ */
+function civihr_employee_portal_update_7034() {
+  features_revert(['civihr_employee_portal_features' => ['page_manager_pages']]);
+  features_revert(['civihr_employee_portal_features' => ['filter']]);
+  civicrm_initialize();
+  drush_civihr_employee_portal_refresh_node_export_files();
+}
+
+/**
+ * Revert emergency contacts view
+ */
+function civihr_employee_portal_update_7035( ) {
+  $view = views_get_view('emergency_contacts');
+  $view->delete();
```

Comment:
@powdevel why are you deleting view here ?

Reply:
@hitesh-jain See what drush does to revert a view. I tested this code.
https://api.drupal.org/api/views/drush%21views.drush.inc/function/views_revert_view/7.x-3.x

Reply:
@powdevel use feature_revert function to revert view if the view is added using feature or use drush_invoke function. why we are not using any of this ?

Reply:
d@hitesh-jain I didn't know about that function, and know that I know it exists, the function doesn't exists when executing my code. Maybe it is obvious but I can't figure out a way to include this function safely, currently I asked this as a question here https://drupal.stackexchange.com/questions/264375/where-is-drush-invoke-function-for-drupal-7

Reply:
@powdevel so we can't use drush_invoke here ?